XRDXRFutils/spectra.py

```diff
@@ -78,6 +93,15 @@ def from_spe(self, filename):
                 warnings.warn(f'Unknown data format in file \'{filename}\'')
                 return None
 
+    @property
+    def energy(self):
+        return self.fce_calibration(self.channel, *self.opt)
```

Comment:
Isnt opt part of Calibration class?
I do not see opt defined in Spectra nor SpectraXRF